### PR TITLE
Theyalow

### PR DESCRIPTION
**Pull request by o1vanova**
Web page for task [Theyalow](https://github.com/rolling-scopes-school/tasks/blob/master/tasks/markups/level%201/theyalow/theyalow-en.md)

Browser Support: Google Chrome, Mozilla Firefox, Microsoft Edge.
[Desktop version](https://o1vanova.github.io/theyalow/)

### CODE REVIEW

**Comment by katsu223** [comment]
The most common mistake is `0.5px`, instead of `.5px` in your css properties. Another mistake is usage of `height` when you can do the same with paddings.

File: style.css [hunk 1]
```diff
@@ -1,940 +1,951 @@
...
+  width: 100%;
+  background: url('./assets/images/banner/shadow.png');
+}
+
+.banner .content-container {
+  justify-content: space-between;
+}
+
+.block {
+  border-left: 1px solid rgba(219,219,219);
```
Comment (katsu223) [inline style.css]:
```suggestion
  border-left: 1px solid rgba(219, 219, 219);
```

File: style.css [hunk 2]
```diff
@@ -1,940 +1,951 @@
...
+  flex: 1;
+}
+
+.block:last-of-type {
+  border-right: 1px solid rgba(219,219,219);
```
Comment (katsu223) [inline style.css]:
```suggestion
  border-right: 1px solid rgba(219, 219, 219);
```

File: style.css [hunk 3]
```diff
@@ -1,940 +1,951 @@
...
+  position: absolute;
+}
+
+.scrollbar__item--left {  
+  left: 0;
+}
+
+.scrollbar__item--right {  
+  right: 0;
+}
```
Comment (evgenydus) [inline style.css]: Not needed if you use flex on parent.

File: style.css [hunk 4]
```diff
@@ -1,940 +1,951 @@
...
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+}
+
+.scrollbar__item {
+  width: 32px;
+  height: 57px;
```
Comment (evgenydus) [inline style.css]: Use paddings with `svg` content instead of pseudo-.

**Pull request merged**